src/main/resources/db/migration/V1_2__create_media_request_table.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE IF NOT EXISTS media_request (
+  request_id                integer                     NOT NULL
+, case_id                   text                        NOT NULL
+, requester                 text                        NOT NULL
+, start_time                timestamp without time zone NOT NULL
+, end_time                  timestamp without time zone NOT NULL
+, request_type              text                        NOT NULL
+, status                    text                        NOT NULL
+, attempts                  integer                     NOT NULL DEFAULT 0
+, outbound_location         text                        NULL
+, media_type                text                        NULL
+, output_filename           text                        NULL
+, last_accessed_date_time   timestamp without time zone NOT NULL
+, created_date_time         timestamp without time zone NOT NULL
+, last_updated_date_time    timestamp without time zone NOT NULL
+, CONSTRAINT audio_request_pkey PRIMARY KEY (request_id)
```

Comment:
I guess this constraint should be renamed to be `media_request_pkey`.
But notice that Barry now has v23 `PRIMARY KEY (moj_mer_id)`

https://cgigbr.sharepoint.com/teams/PRJ00013693/Shared%20Documents/Forms/All%20Documents.aspx?id=%2Fteams%2FPRJ00013693%2FShared%20Documents%2FGeneral%2FDesigns%2FBarry%2Fmodernised%5Fdarts%5Fschema%5Fv23%2Esql&parent=%2Fteams%2FPRJ00013693%2FShared%20Documents%2FGeneral%2FDesigns%2FBarry

```
CREATE TABLE DARTS.moj_media_request
(moj_mer_id                 INTEGER                    NOT NULL
,moj_hea_id                 INTEGER                    NOT NULL
,requestor                  CHARACTER VARYING
,request_status             CHARACTER VARYING
,req_proc_attempts          SMALLINT 
,start_ts                   TIMESTAMP WITH TIME ZONE
,end_ts                     TIMESTAMP WITH TIME ZONE
,created_ts                 TIMESTAMP WITH TIME ZONE
,last_updated_ts            TIMESTAMP WITH TIME ZONE
,last_accessd_ts            TIMESTAMP WITH TIME ZONE
,output_filename            CHARACTER VARYING
,output_format              CHARACTER VARYING
);
```